src/client/client.h

```diff
@@ -500,11 +500,8 @@ class Client : public con::PeerHandler, public InventoryManager, public IGameDef
 	u8 m_server_ser_ver;
 
 	// Used version of the protocol with server
-	// Values smaller than 25 only mean they are smaller than 25,
-	// and aren't accurate. We simply just don't know, because
-	// the server didn't send the version back then.
 	// If 0, server init hasn't been received yet.
-	u8 m_proto_ver = 0;
+	u16 m_proto_ver = 0;
```

Comment:
Non-functional change since an u16 is currently downcasted to this u8 version number on init.